src/test/kotlin/no/nav/lydia/ia/sak/api/HendelseSerialiseringTest.kt

```diff
@@ -0,0 +1,45 @@
+package no.nav.lydia.ia.sak.api
+
+import com.github.guepardoapps.kulid.ULID
+import io.kotest.matchers.booleans.shouldBeTrue
+import io.kotest.matchers.types.shouldBeTypeOf
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import no.nav.lydia.helper.SakHelper.Companion.toJson
+import no.nav.lydia.ia.årsak.domene.BegrunnelseType
+import no.nav.lydia.ia.årsak.domene.ValgtÅrsak
+import no.nav.lydia.ia.årsak.domene.ÅrsakType
+import no.nav.lydia.ia.sak.domene.IASakshendelse
+import no.nav.lydia.ia.sak.domene.SaksHendelsestype.VIRKSOMHET_ER_IKKE_AKTUELL
+import no.nav.lydia.ia.sak.domene.VirksomhetIkkeAktuellHendelse
+import kotlin.test.Test
+
+class HendelseSerialiseringTest {
+
+
+    @Test
+    fun `skal kunne dekode en serialisert sakshendelse som en VirksomhetIkkeAktuellHendelse når payload er gyldig`() {
```

Comment:
Her ser det ut som man tester kotlinx-serialization. Er det meningen? Isåfall ser jeg ingen nytteverdi av den. Ser heller nytte av å ha en test med ugyldig payload for å se at man får feilmelding